api/src/test/java/uk/gov/dhsc/htbhf/eligibility/testhelper/TestConstants.java

```diff
@@ -0,0 +1,26 @@
+package uk.gov.dhsc.htbhf.eligibility.testhelper;
+
+import java.math.BigDecimal;
+import java.time.LocalDate;
+
+public final class TestConstants {
+
+    public static final LocalDate DOB = LocalDate.parse("1985-12-31");
```

Comment:
might be better to prefix these with LISA_ to be explicit about the fact that they all relate to a single person?

Reply:
I also think its not a good idea to mix DOB / date of birth as its just another possible point of confusion. As we use `dateOfBirth` in the code I suggest this should be `LISA_DATE_OF_BIRTH`.